src/application.cpp

```diff
@@ -33,6 +33,12 @@
 #include <QLibraryInfo>
 #include <QSysInfo>
 
+#ifndef DISABLE_GUI
+#ifdef Q_OS_MAC
+#include <QFont>
+#endif // Q_OS_MAC
+#endif // DISABLE_GUI
```

Comment:
Sorry I didn't notice this before. Can you remove the comments here? They aren't really needed.